services/datamanager/builtin/builtin.go

```diff
@@ -648,6 +649,13 @@ func (svc *builtIn) uploadData(cancelCtx context.Context, intervalMins float64)
 	})
 }
 
+func isOffline() bool {
+	timeout := 5 * time.Second
+	_, err := net.DialTimeout("tcp", "viam.com:443", timeout)
```

Comment:
This is pinging prod, is there an easy way to get the right remote app link, if its staging or something else?

I know this is a hotfix so not super important. Also I guess if it can't ping prod it can't ping the other environments. This could really be any URL. So probably fine. I've talked myself out of it now, but just wanted to call this out.